app/src/main/java/com/example/harvestfresh/ProductListingAdapter.java

```diff
@@ -17,11 +24,21 @@
 
 public class ProductListingAdapter extends RecyclerView.Adapter<ProductListingAdapter.ViewHolder> {
     public static final String CART_ADD = "An Item has been added";
+    private static final String CHANNEL_ID = "HarvestFresh";
+    private static final String CHANNEL_NAME = "HarvestFreshChannel";
+    private static final String CHANNEL_DESCRIPTION = "Channel for Alarm Manager";
+    private static final long ALARM_TIME = 60000;
```

Comment:
Make this variable a little more descriptive: For example, this is a count in MS, so its good to capture that in the variable name, such as "ALARM_TIME_MS". But I also notice that this const doesn't hold a time, it holds a timespan, so it would be more accurate to call this something like ALARM_TIME_INTERVAL_MS, or ALARM_WAITING_PERIOD_MS or something.

The next thing is that for something like this, the number here is constructed...1000 ms in a second, times 60 seconds in a minute. You can make that clearer by doing "60 * 1000" which also helps reduce the risk of putting too many or not enough zeros.

Now, this is a special case: Consts related to time are really common, so Java (and most other languages) includes some utils for this. See here for idas on doing this:
https://stackoverflow.com/questions/2442486/time-consts-in-java
I like "DateUtils.MINUTE_IN_MILLIS".